packages/flutter/test/cupertino/route_test.dart

```diff
@@ -2691,6 +2691,54 @@ void main() {
     await tester.pumpAndSettle();
   });
 
+  testWidgets('fullscreen routes do not transition previous route', (WidgetTester tester) async {
+    await tester.pumpWidget(
+      CupertinoApp(
+        initialRoute: '/',
+        onGenerateRoute: (RouteSettings settings) {
+          if (settings.name == '/') {
+            return PageRouteBuilder<void>(
+              pageBuilder: (_, __, ___) {
```

Comment:
Nit: [Avoid anonymous parameter names](https://github.com/flutter/flutter/blob/master/docs/contributing/Style-guide-for-Flutter-repo.md#avoid-anonymous-parameter-names), though I know we don't always follow this in tests...

Also a few other instances below.